src/sysfiles.h

```diff
@@ -82,6 +82,24 @@ extern Int SyLoadModule( const Char * name, InitInfoFunc * func );
 
 /****************************************************************************
 **
+*F * * * * * * * * * * finding location of executable * * * * * * * * * * * *
+*/
+
+/* On some OSes PATH_MAX is unreliable, so we make our buffer longer */
+
```

Comment:
Actually that is an understatement. PATH_MAX is basically useless on every system. Any hard coded buffer size is potentially to small.